packages/dashboard/src/components/pageStructure/index.js

```diff
@@ -190,35 +242,57 @@ export function LeftRail() {
         </Content>
         <Content>
           <NavList>
-            {enabledPrimaryPaths.map(({ Icon, ...path }) => (
-              <NavListItem key={path.value}>
-                <NavLink
-                  active={path.value === state.currentPath}
-                  href={resolveRoute(path.value)}
-                  size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
-                  isBold
-                  aria-label={
-                    path.value === state.currentPath
-                      ? sprintf(
-                          /* translators: %s: the current page, for example "My Stories". */
-                          __('%s (active view)', 'web-stories'),
-                          path.label
-                        )
-                      : path.label
-                  }
-                  {...(path.isExternal && {
-                    rel: 'noreferrer',
-                    target: '_blank',
-                    onClick: (evt) => onExternalLinkClick(evt, path),
-                  })}
-                >
-                  {Icon && <Icon width="22px" />}
-                  <PathName as="span" isBold>
-                    {path.label}
-                  </PathName>
-                </NavLink>
-              </NavListItem>
-            ))}
+            {enabledPrimaryPaths.map(({ Icon, ...path }) => {
+              const isNotificationBubbleEnabled =
+                path.value === APP_ROUTES.TEMPLATES_GALLERY;
+              const appendNewBadgeToLable = (label) =>
+                isNotificationBubbleEnabled
+                  ? sprintf(
+                      /* translators: 1: current page. 2: number of new items. */
+                      __('%1$s (%2$s new)', 'web-stories'),
```

Comment:
Since the number of new templates could be 1 or many, we should probably use `_n()` here.

Reply:
does it need `_n()` ? cus both options would be the same for singular and plural here. Maybe not the case for other languages tho? 🤷

Reply:
Yeah I was thinking of other languages here. English rarely needs any special handling, the language is too simple 😄 

But after thinking about it again I think the current `__()` should be OK. So let's leave it